tsconfig.json

```
@@ -0,0 +1,15 @@
+{
+  "compilerOptions": {
+    "checkJs": true,
```

```suggestion
    "checkJs": false,
```

After renaming `responseStatuses` to ts